src/pretix/control/views/organizer.py

```diff
@@ -1252,6 +1253,7 @@ class WebHookLogsView(OrganizerDetailViewMixin, OrganizerPermissionRequiredMixin
     def get_context_data(self, **kwargs):
         ctx = super().get_context_data(**kwargs)
         ctx['webhook'] = self.webhook
+        ctx['retry_count'] = self.webhook.retries.count()
```

Comment:
Should we maybe add `retry_celery_cutoff` to ctx and use it to tell the minutes in webhook_logs.html? Probably not worth the effort (`retry_celery_cutoff` would need to be moved outside of `send_webhook`), but would help in the future if/when we change the cutoff-time for celery-based retries.

```suggestion
        ctx['retry_count'] = self.webhook.retries.count()
        ctx['retry_cutoff_minutes'] = retry_celery_cutoff // 60
```

Reply:
I mostly put it into variable in the template so if we change it, translations do not need to change. I don't think it's worth moving it to a global variable for now.